### DISCUSSION TITLE
SlingerPlayer on Win.7:  remote control window is "empty"

### DISCUSSION THREAD

**Discussion by bradut** [GerryDazoo/SlingerPlayer#19] [General]
Hi,
SlingerPlayer for Windows runs fine on my machines [Win7, Win10].

**The issue:**
I installed SlingerPlayer into someone's Windows 7 laptop and, while streaming is fine, the remote control window is empty, regardless of what SlingBox is selected.
I've right-clicked on that window hoping the be able to inspect its HTML content, but no luck.
(As an workaround, I've placed the URLs of the remotes as shortcuts on that machine's desktop window.)

Aside from the remote issue, the app plays streams fine and is able to select different SlingBoxes.

Any ideea how can I solve this?

![SlingerPlayer Cannot Load Remote HTML Form](https://user-images.githubusercontent.com/2496596/235352691-e25a5c81-7744-4cb7-9af1-a0e81e5dda32.jpg)

**Comment by GerryDazoo** [comment]
give this a try
https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section

**Reply by bradut** [reply]
It works !
Thank you Gerry !

For the record: out of the three options on that page, the one that helped me was: **Evergreen Bootstrapper**
